contracts/RLPEncode.sol

```diff
@@ -0,0 +1,77 @@
+// Library for RLP encoding a list of bytes arrays.
+// Modeled after ethereumjs/rlp (https://github.com/ethereumjs/rlp)
+// [Very] modified version of Sam Mayo's library.
+pragma solidity ^0.4.23;
+import "./BytesLib.sol";
+
+library RLPEncode {
```

Comment:
The file is imported from another repo.
Do we want to update comments as per Natspec?
cc: @pgev

Reply:
We have a separate ticket for this. MOSA-135